util/Migrator/DbScripts/2023-03-10_00_OrganizationUserReadByUserIdWithPolicyDetails.sql

```diff
@@ -0,0 +1,35 @@
+CREATE PROCEDURE [dbo].[OrganizationUser_ReadByUserIdWithPolicyDetails]
```

Comment:
In case the sproc already exists (always need to do this in migrations):

```suggestion
CREATE OR ALTER PROCEDURE [dbo].[OrganizationUser_ReadByUserIdWithPolicyDetails]
```